Automattic/wp-calypso
# Stats: Adding skip button functionality

Related to #86136

## Proposed Changes

* adding the following skip button functionality
   * adding a mutation that updates `focus-jetpack-purchase`
   * triggering an event before redirecting to the main page
   * redirecting to the main page

## Testing Instructions



* create a JN blog and connect Jetpack
* open the live branch and navigate to the Stats page for the new blog
* apply `stats/checkout-flows-v2` feature flag and verify that you are redirected to a purchase page
* click on `I'll do it later` button and verify in the network tab that a call was made to update the nieces and a Tracks event was triggered
* when you navigate or you are redirected to the `Traffic` page you will still end up on the purchase page. This is due to react-query caching and will be addressed after #86625 is merged

## Pre-merge Checklist
